src/components/mainpage.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+import CircleIcon from '@mui/icons-material/Circle';
+import FacebookRoundedIcon from '@mui/icons-material/FacebookRounded';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import { Splide, SplideSlide } from '@splidejs/react-splide';
+import '@splidejs/react-splide/css';
+import yatches from './dummydata/joydata';
+import './mainpage.css';
+
+const MainPage = () => (
+  <div>
+    <Wrapper>
+      <h2>Popular Destinations</h2>
+      <h3> Select your desired destination</h3>
+      <div className="dots">
+        {[...Array(15)].map(() => (
+          <div key={uuidv4()}>
+            <CircleIcon color="disabled" sx={{ fontSize: '0.5rem' }} />
+          </div>
+        ))}
+      </div>
+      <Splide
+        options={{
+          perPage: 3,
+          arrows: true,
+          pagination: false,
+          drag: 'free',
+          gap: '1rem',
+        }}
+      >
+        {yatches.map((item) => (
+          <SplideSlide key={item.id}>
+            <Card>
+              <div className="card">
+                <img src={item.image} alt={item.name} />
+                <div className="yatch_name">
+                  <h4>{item.name}</h4>
+                </div>
+                <div className="dots">
+                  {[...Array(15)].map(() => (
+                    <div key={uuidv4()}>
+                      <CircleIcon
+                        color="disabled"
+                        sx={{ fontSize: '0.7rem' }}
+                      />
+                    </div>
+                  ))}
+                </div>
+                <p>{item.description}</p>
+                <div className="social">
+                  <FacebookRoundedIcon
+                    sx={{
+                      fontSize: '1.5rem',
+                      color: 'rgb(0, 0, 0, 0.26)',
+                      marginRight: '1rem',
+                    }}
+                  />
+                  <InstagramIcon
+                    sx={{
+                      fontSize: '1.5rem',
+                      color: 'rgba(0, 0, 0, 0.26)',
+                      marginRight: '1rem',
+                    }}
+                  />
+                  <TwitterIcon
+                    sx={{ fontSize: '1.5rem', color: 'rgb(0, 0, 0, 0.26)' }}
+                  />
+                </div>
+              </div>
+            </Card>
+          </SplideSlide>
+        ))}
+      </Splide>
+    </Wrapper>
+  </div>
+);
+
+const Wrapper = styled.div`
+  margin: 4rem 0;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
I think it would be good to avoid putting style in the js file. It's a better practice in my opinion to have a more maintainable code. Then kindly move this on style file